src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -460,7 +460,7 @@ private void HandleChangesForStaticIndexes(DatabaseRecord record, long index, Li
                     }
 
                     var configuration = new FaultyInMemoryIndexConfiguration(_documentDatabase.Configuration.Indexing.StoragePath, _documentDatabase.Configuration);
-                    var fakeIndex = new FaultyInMemoryIndex(exception, indexName, configuration, definition);
+                    var fakeIndex = new FaultyInMemoryIndex(exception, indexName, configuration, definition, SearchEngineType.None);
```

Comment:
We can have the search engine type config option available in `definition`. So maybe we could try to extract it and pass here

Reply:
Can we?
In the case of `AutoIndexes` it would take DB default value (but index can be done on previous value etc). 
I added it for StaticIndexes.

Reply:
okay